onnx/backend/test/case/node/globalmaxpool.py

```diff
@@ -24,7 +24,7 @@ def export():  # type: () -> None
         )
         x = np.random.randn(1, 3, 5, 5).astype(np.float32)
         spatial_shape = np.ndim(x) - 2
-        y = np.max(x, axis=tuple(range(spatial_shape, spatial_shape + 2)))
+        y = np.max(x, axis=tuple(range(2, spatial_shape + 2)))
```

Comment:
Thanks for the fix. Is it possible to use the ```keepdims``` parameter of ```np.max``` and eliminate the code below?

Reply:
Yes to both - done!